docs/how-to/watch-cou-logs.rst

```diff
@@ -0,0 +1,48 @@
+==============
+Watch COU logs
+==============
+
+Below is the script to watch the COU logs when doing the development.
+This script watches every log file in the directory, even newly created files.
+
+.. code:: bash
+
+    #!/bin/bash
+
+    # Change LOG_DIR to the target directory
+    DIRECTORY="$LOG_DIR"
```

Comment:
Isn't this a standard location?  Can we hardcode this to `~/.local/share/cou/logs/*`, so by default users who copy/paste the script don't need to make changes?

Reply:
Updated

Reply:
Ah sorry I realised the path I suggested wasn't correct, but you got it right, thanks.